common/src/envs/constants.ts

```diff
@@ -33,7 +33,9 @@ export function isManifoldId(userId: string) {
   return userId === 'IPTOzEqrpkWmEzh6hwvAyY9PqFb2'
 }
 
-export const DOMAIN = ENV_CONFIG.domain
```

Comment:
Would add a note about when to use this, e.g. "// Use this instead of BASE_URL for /backend code and when you want the domain without the protocol".

Reply:
to be clear, I am not sure that you absolutely need to use it for /backend code, as opposed to BASE_URL. I am just more sure this won't break anything